include/PROPOSAL/math/MathMethods.h

```diff
@@ -171,7 +172,13 @@ std::pair<double, double> welfords_online_algorithm(double& newValue, unsigned i
 /// @return         Sampled value
 
 double SampleFromGaussian(double mean, double sigma, double rnd,
-                          double min = -std::numeric_limits<double>::infinity(),
-                          double max = std::numeric_limits<double>::infinity());
+                          double min = -INF,
+                          double max = INF);
```

Comment:
I think for `INF` the `<cmath>` is required.

Reply:
Also what is wrong with using the modern c++ version?

Reply:
INF actually is PROPOSAL::INF, where it is defined as `std::numeric_limits<double>::infinity()`

Reply:
sorry i had overlooked it and thought it was a c macro.